SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1612,6 +1612,25 @@ void USpatialNetDriver::ProcessRPC(AActor* Actor, UObject* SubObject, UFunction*
 
 	if (IsServer())
 	{
+		if (PackageMap->GetEntityIdFromObject(CallingObject) == SpatialConstants::INVALID_ENTITY_ID)
+		{
+			AActor* TargetActor = Cast<AActor>(CallingObject);
+			if (TargetActor == nullptr)
+			{
+				TargetActor = Cast<AActor>(CallingObject->GetOuter());
```

Comment:
I think Actor is always != nullptr here, which probably means we can just use Actor, rather than than needing this logic to get an actor from a subobject?

Reply:
Is there not a possible case where this function is called on a subobject and the subobject is not an Actor, then TargetActor would be null?

Reply:
@m-samiec ?

Reply:
Yeah I'm not sure it's possible for Actor to be nullptr here, so should be safe to use directly 👍

Reply:
Great, thanks for the confirmation. I have removed these few lines.